src/sdk/pynni/nni/hyperband_advisor/hyperband_advisor.py

```diff
@@ -214,6 +208,14 @@ def _record_hyper_configs(self, hyper_configs):
         self.num_configs_to_run.append(len(hyper_configs))
         self.increase_i()
 
+def extract_scalar_reward(value, scalar_key='default'):
+    if isinstance(value, float) or isinstance(value, int):
```

Comment:
what's the difference between this function and the extract_scalar_reward function in the other file?

Reply:
the same.